### PR TITLE
Project Movies- Sara Stjernberg

### PR DESCRIPTION
**Pull request by stjernberg**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/PopularMovies.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            .then(json => setMovies(json.results))
+           }, [])
+
+    return (        
+         <section className="movies-list">
+            {movies.map((movie) => ( 
+               <MoviePoster 
+                 key={movie.id} 
+                  {...movie} 
+                  />
```
Comment (petrame) [inline code/src/components/PopularMovies.js:23]: Nice that MoviePoster got it's own component! It looks so much cleaner and easy to read than my own movie list.

File: code/src/urls.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const API_KEY = "fe9bb5c09e9066da8266038ab1a77a26"
+export const POPULAR_URL = `https://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}&language=en-US&page=1`
+export const UPCOMING_URL = `https://api.themoviedb.org/3/movie/upcoming?api_key=${API_KEY}&language=en-US&page=1`
+export const NOWPLAYING_URL = `https://api.themoviedb.org/3/movie/now_playing?api_key=${API_KEY}&language=en-US&page=1`
+export const TOPRATED_URL = `https://api.themoviedb.org/3/movie/top_rated?api_key=${API_KEY}&language=en-US&page=1`
```
Comment (petrame) [inline code/src/urls.js:5]: ⭐

**Review by petrame — APPROVED** [review]
Nice movie site Sara 🤩
Especially the feature that the headlines change color when you click on them, because I myself did not manage to do that.
It's easy to read your code since it well structured and named.
I could see that you have the same error message in the console as me, but I haven't solved it myself so I can't give any good advice there 😀

File: code/src/components/Nav.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom';
+
+import '../styles/Nav.css'
+import MovieIcon from 'icon/icon.png'
```
Comment (EmOlsz) [inline code/src/components/Nav.js:5]: Uppercase names are reserved for components only, not images. Should rather be :

```
import movieIcon from 'icon/icon.png'
```

File: code/src/pages/MovieDetails.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+export const MovieDetails = () => {
+    const { id } = useParams()
+    const MOVIE_URL = `https://api.themoviedb.org/3/movie/${id}?api_key=${API_KEY}&language=en-US`
+    const [movie, setMovie] = useState({})
+
+    useEffect(() => {
+        fetch(MOVIE_URL)
+            .then(response => response.json())
+            .then((json) => {
+                console.log(json)
```
Comment (EmOlsz) [inline code/src/pages/MovieDetails.js:16]: We should remove all console.logs before final push

**Pull request closed without merging**